libr/anal/p/anal_8051.c

```diff
@@ -881,6 +881,10 @@ static ut32 map_direct_addr(RAnal *anal, ut8 addr) {
 static int i8051_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *buf, int len, RAnalOpMask mask) {
 	set_cpu_model (anal, false);
 
+	if (mask & R_ANAL_OP_MASK_ESIL) {
```

Comment:
Not the same. Thats wrong

Reply:
I have concluded that we don't actually have a MASK_VAL implementation yet.